### ISSUE TITLE
[Sybil Report]

### ISSUE THREAD

**Issue by TessyBach** [LayerZero-Labs/sybil-report#7246]
**1. Reported Addresses**
0x4244776ad1cb7f788523bc9be91ca436c67506b0
0x0c35aba28d30ceedf2276ea1629b1d6ac7d76d4e
0xd540f4fe3b96695e323e3907c2b8cf8655e0244c
0xaf25aa29ccb335c8667e1708d41e931688ea81e0
0xac7e1fb507d69aca43733f0b413a7919f4bfc9b8
0x5ae09483b7e1261d30af0376a8cd8e6ca1aeaa43
0xf2b48eaec048127ecca812302fc7924da1dbd58e
0x82181e92586d64f10454ec64f9fa5ed7275959d8
0x34581e2c9cab87e545a147733d7a0a2006c9ec31
0x3f91979e4d4e2b0f4e6860b6d7f1d2bc2b618f46
0xefa68de905f5a06e5cbeb1544189b227f9dc7c5e
0x6b6225eab3b3561c14a3813b920ed327c48c2321
0x4474700c0478deb2e4b3c729057cb41c38d501a8
0xaf7cf990b1ad10043d4a64af59c5d657e9c4de29
0x8af4de28c3ba4f130a02b4152bdce4e08a7ad3f2
0x5a95f870da56882ff27940d1f08d829e6d2f6a1e
0x37ed6f830e207fc0725dc4aa35cffacf012020f2
0x9419530ad4d7bf421d374467de46b02b1cbc4c47
0x340e9d6da2ab8f8bac32a35fb7139c459a2013bc
0x029d29c07e49574f54d55c7eadcdcf487f3a03e9
0x7dcd3037ac2b122af0acf6a0f1aa86859f7b9465
0xf05075490beeb1c3450cc8cc7edda298b8f82a5a
0x209fbadc1dd19067cf3e79b5c724512272a79835
0x66bf0a5cdd69e88e7cde20eec5e82427c82b7c22
0x2c7e17195182616cfa31bb38a5f97cc052fb0a75
0x794aa80282e23d8f831ac5d3d33bfd1c9ae25bf3
0x4438ba15b226c50e5a128111b94e6a132f6b7ecd
0xe0fe4a091e0336cc0350d61b9e5b46eef0cc9b4d
0xbd4e80e522beecfe0b8c8b2c970c9cfff8054f2e
0x92bc9d813eeb8b4711a0a1be61b62d9586f1d592
0x8dffe11a39baeb696f50dc9002a1a722583be842
0xcf5a3096503763b45564a61585af5ea2cf7e6116
0xfc7ddcb69edbf6f88668e377e359ede407b80699
0x9adbba6526eba480adc28d93a2d1b2cf63795990
0xe6da940b94d158690b06f717be19ac7626ee050c
0x3dedbc18b1a9e8c8e521657b8a2be05c9609713f
0x376930fc74c307eb9b47dae3514a16f967e7f33e
0x8029c32587b8558df689a0560e3ccfb00122e7a2
0x733e824d7e4433e8490c605b3cdd9929aa1a1d7c
0x934a7577f93663bdb9efc9680a6dee0e4a951cc1
0x3011f9a01f2399cd60c5e24c34e4ab58d8b85394
0x58f4f5fef0fc962e5706819179476194ce2749bb
0x7109ba953980e8d0902bdea496772c220a7f2a70
0x34e5e9707730a13d498a9a745df0f546be6af294
0xad832566f3a8c68e6d9d27ebad828af535d4a932
0x7940a8144f5f5f1e1e1dc8d073bca6afca51922c
0xb5a33e6fa0ba6af5d83cb1711f0c1d5d2a29391e
0x441467e499329672f0658e5663e39d7927787a11
0x926d4c19094f7c7a3349ec53fdde375117fb3dc3
0x4d87c70a7fa79f1b97761e30dcdb4283fd972ccf
0x4d87c70a7fa79f1b97761e30dcdb4283fd972ccf
0x1e0aae0c9813b0a957b146411038d94ccaaaa655
0x9964074f1e302325b4faeb5ce755cbcfa91e07cb
0xee6e78eb308d3d6e98993d5d192308cc75ade98b
0x6944fc1eeee9f5c789d38a6dfc9a0b5d7d14bf69
0x43d94cdefc092abda6dacfb6980f3c35a7789f1a
0x387c15a2f8c008ac737061d078b6195cedb9c4cb
0x954054abdbdead4f4bc6d857fb6925fbd744437e
0x4e9d1e1bcbd292a533a67ad4cfc29a95adde86ef
0x58117cd74c97ff72a912ab21cebb351dd5eafd9c

**2.  Description**
Airdrop from AEVO. AEVO tokens were sent to the same address from all of these wallets, indicating that they were the same person's accounts. All proofs and hash transaction data are presented in EXCEL format.
[AEVO transfer sibil.xlsx](https://github.com/LayerZero-Labs/sybil-report/files/15365135/AEVO.transfer.sibil.xlsx)

**3. Reward Address (If Eligible)**
0xF776b567BB25f4E3a8EfD56Da1a3924e24Ff521F

**Comment by evilpekarr** [comment]
This is not Sybil's wallet, they are all have been stolen earlier this year (January). One of my main wallet's in this list. Scamers are draining all of them, that's why money flows on the one adress.

**Comment by TessyBach** [comment]
Yeah, these people branded the drop and sent it to one address, which makes them sybils, but what you write is just trying to save your accounts

**Comment by TessyBach** [comment]
Linking more than one wallet is Sybil

**Comment by evilpekarr** [comment]
Dude, I'm not trying to save wallet because it was stollen. If u are so smart, u can check it how i was drained at the end of January. All my money was stollen and I'm not hopping to recieve something. Better to ban this accounts so thiefs won't able to recieve airdrop.
![image](https://github.com/LayerZero-Labs/sybil-report/assets/101177946/7bc3cd9c-0333-40fc-b13a-f0acd9a80f43)

**Comment by evilpekarr** [comment]
I am a blogger and wrote about this in my community, trying to draw attention to it. The theft hit me very hard. Now I'm trying to somehow get out of this unpleasant situation.
